tests/e2e/search.spec.js

```diff
@@ -44,9 +44,8 @@ test('check search works', async ({page, requestUtils}) => {
     const searchTags = await page.locator('.search-result-item-tag').allInnerTexts();
 
     expect(searchResult).toContain(testId);
-    expect(searchPage).toContain(tagPageTitle);
     expect(searchPage).toContain(postTitle);
-    expect(searchTags).toContain(`#${tagName}`);
+    expect(searchTags).toContain(tagPageTitle);
```

Comment:
Maybe we don't need this variable? It's only called once, can't we directly use `#${tagName}`?